browser/src/Services/QuickOpen.ts

```diff
@@ -40,6 +38,13 @@ export class QuickOpen {
     public show(): void {
         const overrriddenCommand = Config.getValue<string>("editor.quickOpen.execCommand")
 
+        UI.showPopupMenu("quickOpen", [{
```

Comment:
I logged #269 to build on this and better support async in general with Ctrl-P - it'd be nice to have a gradual loading experience, where you can search results as they are being populated.